packages/cli/src/commands/network/parameters-l2.test.ts

```diff
@@ -0,0 +1,94 @@
+import { testWithAnvil } from '@celo/dev-utils/lib/anvil-test'
+import { setupL2 } from '../../test-utils/chain-setup'
+import { stripAnsiCodesFromNestedArray, testLocallyWithWeb3Node } from '../../test-utils/cliUtils'
+import Parameters from './parameters'
+
+process.env.NO_SYNCCHECK = 'true'
+
+testWithAnvil('network:parameters', (web3) => {
+  test('runs', async () => {
+    await setupL2(web3)
+    const spy = jest.spyOn(console, 'log')
+    await testLocallyWithWeb3Node(Parameters, [], web3)
+    expect(stripAnsiCodesFromNestedArray(spy.mock.calls)).toMatchInlineSnapshot(`
+      [
+        [
+          "Attestations: 
```

Comment:
this test will be updated when we handle the changes to network:parameters which Leszek is working on